### DISCUSSION TITLE
🐛 [BUG] Pie/Column/Bar not render correctly when field name contains asterisk character

### DISCUSSION THREAD

**Discussion by lessmost** [antvis/G2Plot#3590] [Q&A] [answered]
### 🐛 Bug description [Please make everyone to understand it]

Pie/Column/Bar not render correctly when field name contains asterisk character
![image](https://user-images.githubusercontent.com/1142242/224208828-4a4670f4-fb70-4217-9396-2889a046a383.png)

> Please provide a link by forking these links [G2Plot](https://codesandbox.io/s/g2plot-issue-template-gzfdq?file=/index.ts) or GitHub repo, a minimal reproduction.

* _Required_ **Link to minimal reproduction**:

https://codesandbox.io/s/serene-nightingale-tpzq01?file=/index.ts

### 🚑 Any additional [like screenshots]

* **G2Plot Version**:
* **Platform**:

**Comment by hustcc** [comment, marked as answer]
`*` 在 G2 4.x 中属于是关键词，在代码中会用于分割：

```ts
import { Chart } from '@antv/g2';

const chart = new Chart();

chart
  .interval()
  .position('x*y') // 👈🏻 这里

chart.render();
```

在 4.x 中无解，这个在 [G2 5.0](https://g2.antv.antgroup.com/) 中默认已经解决了。

![image](https://github.com/antvis/G2Plot/assets/7856674/7ec44a10-5e0b-4867-b057-287697d84216)
